src/client/editor/Editor.js

```diff
@@ -1858,33 +1862,53 @@ export default class Editor {
     this.history.redo();
   }
 
-  getSceneAttribution() {
-    const attributions = new Set();
+  getSceneContentAttributions() {
+    const contentAttributions = [];
```

Comment:
Nit: You can probably just make `contentAttributions` a Set instead of having both an array and a set. If you want to return an array so can use all the array methods, you can use `return Array.from(contentAttributions)`